src/ontology/mondo-edit.obo

```diff
@@ -296560,12 +296560,12 @@ synonym: "ALA dehydratase deficiency pophyria" RELATED [GARD:0004445]
 synonym: "ALAD deficiency" RELATED [OMIM:612740]
 synonym: "ALAD Porphyria" EXACT [NORD:747]
 synonym: "ALAD porphyria" EXACT [Orphanet:100924]
+synonym: "ALAD-related porphyria" EXACT CLINGEN_LABEL []
 synonym: "aminolevulinate dehydratase deficiency porphyria" RELATED [GARD:0004445]
 synonym: "Delta-aminolevulinate dehydratase deficiency" RELATED [OMIM:612740]
 synonym: "Doss porphyria" RELATED [OMIM:612740]
 synonym: "Lead poisoning, susceptibility to" RELATED [OMIM:612740]
 synonym: "porphobilinogen synthase deficiency" RELATED [OMIM:612740]
-synonym: "porphyria due to ALA dehydratase deficiency" EXACT CLINGEN_LABEL []
```

Comment:
We should not remove a synonym unless it is not correct, because people might still be looking for this disease using this gene. 
Note that this synonym is also a ClinGen label. We are working on setting up a process to make sure that we have only one ClinGen_label per term

Reply:
never mind, this is the same as the term label